packages/elements-core/src/components/Docs/HttpOperation/Responses.tsx

```diff
@@ -86,7 +90,7 @@ const Response = ({ response, onMediaTypeChange }: ResponseProps) => {
 
       {contents.length > 0 && (
         <>
-          <SectionSubtitle title="Body" id="response-body">
+          <SectionSubtitle title={layoutConfig?.responses?.bodyHeader ?? 'Body'} id="response-body">
```

Comment:
In instances like this, can we change to something like 'layoutConfig?.responses?.bodyHeader ?? defaultLayoutConfig.responses.bodyHeader'?  My concern is that we're now managing the english language "defaults" in two spots and would have to keep them in sync.  What would be awesome is if we can assume that layoutConfig is initialized and all it's fields are either set to the default english values or have been overridden - but are always set, so we wouldn't need the ? and ?? notation at all.

Reply:
My intention when writing the code this way was to alter the original code as little as possible, so that the contributors can more easily understand the proposed changes. But I agree, that in the final code the best implementation is the one suggested by you.